exporters/include/opentelemtry/exporters/common/http/httplib.h

```diff
@@ -0,0 +1,8140 @@
+//
+//  httplib.h
+//
+//  Copyright (c) 2020 Yuji Hirose. All rights reserved.
+//  MIT License
+//
+
+#ifndef CPPHTTPLIB_HTTPLIB_H
```

Comment:
How is this library distributed?   

should we "shade" this library behind a namespace so we don't wind up with version issues if someone pulls in a  different version that is somehow not source compatible?